f8a_worker/workers/recommender.py

```diff
@@ -607,6 +608,39 @@ def _get_stack_values(self, similar_stacks_list):
             similarity_list.append(s_stack)
         return similarity_list
 
+def apply_license_filter(epv_list):
+    dependencies = []
+    licenses = []
+    license_score_list = []
+    for epv in epv_list:
+        name = epv.get('pkg', {}).get('name', [''])[0]
```

Comment:
you can move the definition of the `name, version and licenses` into the `license_scoring_input` call, can't you?

Reply:
DONE. I have moved the definitions.